src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -397,6 +397,12 @@ let setup_daemon logger =
   and node_status_url =
     flag "--node-status-url" ~aliases:[ "node-status-url" ] (optional string)
       ~doc:"URL of the node status collection service"
+  and node_error_url =
+    flag "--node-error-url" ~aliases:[ "node-error-url" ] (optional string)
+      ~doc:"URL of the node error collection service"
+  and contact_info =
```

Comment:
what is contact-info supposed to be? An email address? a discord username? Is this something we can share in the doc for this command?

Reply:
I think it could be either email address or discord username. There's no restriction for this fields.

Reply:
I could add a description for this fields saying that both email address or discord username would be allowed.

Reply:
I assume we have something server-side for this, but: do we need to validate that the length of this field isn't too large, to avoid overwhelming the server?